modules/exploits/linux/http/spark_unauth_rce.rb

```diff
@@ -0,0 +1,173 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Apache Spark Unauthenticated Command Execution',
+      'Description'    => %q{
+          This module exploits an unauthenticated command execution vulnerability in Apache Spark with standalone cluster mode through REST API.
+          It uses the function CreateSubmissionRequest to submit a malious java class and trigger it.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'aRe00t',                            # Proof of concept
+          'Green-m <greenm.xxoo[at]gmail.com>' # Metasploit module
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://www.weibo.com/ttarticle/p/show?id=2309404187794313453016'],
```

Comment:
This link returned a blank page for me.

Reply:
![image](https://user-images.githubusercontent.com/22535454/48761879-b3eaac80-ece4-11e8-8958-e89daa06597b.png)

Works for me, have no clue about it.

Reply:
Appears to require authentication? Geolocked?

Reply:
@bcoles I am using the chrome incognito window, works well.
You couldn't access it either?

Reply:
It redirects to `passport.weibo.com`. Web archive does not appear to have the page archived.

Reply:
Yep, I have updated the reference link, it should work now. @bcoles @jrobles-r7 

Weibo seems to have restricted access from IP not in China, such an idiot.

Reply:
New link works for me.